specs/agents/log-correlation.md

```diff
@@ -1,7 +1,41 @@
 ## Log correlation
 
-Agents should provide instrumentation/hooks for popular logging libraries in order to decorate structured log records with trace context. In particular, logging that occurs within the context of a transaction should add the fields `trace.id` and `transaction.id`; logging that occurs within a span should add the fields `trace.id`, `span.id`, and optionally `transaction.id`.
+Agents should provide instrumentation/hooks for popular logging libraries in order to decorate structured log records with trace context.
```

Comment:
[for reviewer] just reformatting, I wonder if we should remove the `span.id` mention as we don't use it as far as I know.

Reply:
No strong feeling on it, but since later parts of this document ignores that field and I'm not aware we'd use it anywhere either, I'd drop it for now.

Reply:
Let's remove it then, we can always re-add it later if we actually need to.